docs/network-iot/run-an-lns/configure-chirpstack.mdx

```diff
@@ -100,4 +100,4 @@ Password: admin
 
 Now that the LNS is operational, proceed to establish routing rules. This ensures that the Helium
 Network accurately directs device packets to the designated LNS. Learn how to implement this in the
-[Routing Rules Configuration Guide](configure-routing-rules).
+[Routing Rules Configuration Guide](/iot/run-an-lns/configure-routing-rules).
```

Comment:
the page is already inside the /iot/run-an-lns/ section so the prefix is not required.